src/main/java/dk/kb/present/holdback/PurposeMatrixSheet.java

```diff
@@ -0,0 +1,134 @@
+package dk.kb.present.holdback;
+
+import dk.kb.util.webservice.exception.InternalServiceException;
+import org.apache.poi.ss.usermodel.Cell;
+import org.apache.poi.ss.usermodel.CellType;
+import org.apache.poi.ss.usermodel.Row;
+import org.apache.poi.xssf.usermodel.XSSFSheet;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+public class PurposeMatrixSheet {
+    private static final Logger log = LoggerFactory.getLogger(PurposeMatrixSheet.class);
+
+    private List<Integer> indholdFra = new ArrayList<>();
+    private List<Integer> indholdTil = new ArrayList<>();
+    private List<String> formNr = new ArrayList<>();
+    private final int NUMBER_OF_COLUMNS = 18; // Number of columns in the sheet
```

Comment:
Can you validate this and throw exception if not correct. Means Excel sheet is out of sync with code.